### DISCUSSION TITLE
Variables: Support formatting __to & __from variables in UTC

### DISCUSSION THREAD

**Discussion by edwardday7** [grafana/grafana#38175] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Allow custom date format to be in Zulu time. Currently if you use `${__to:date}` it will show as zulu time (Ex. 2020-11-09T04:59:00.000Z). However, if you do a custom format by using `${__to:date:YYYY-MM-DD HH-mm Z}` It will show with your local time with the offset (Ex. 2020-11-08 23-59 -05:00). I would be nice to be able to do something like this `${__to:date:iso:YYYY-MM-DD HH-mm}` and it show the time as 2020-11-09 04-59.

**Why is this needed**:
My use case for it is the BigQuery plugin. I'm trying to do the table suffix using custom formatting from the date based on the zulu time (Table suffix is 1109) but it instead does 1108. This would also make things more consistent.

**Comment by torkelo** [comment]
the __to and __from variables will interpolate to epoch millisecond times (in UTC/Zulu)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by markalex2209** [comment]
Maybe alternative format postfix, something like `date_local`, as in `${__to:date_local}` and `${__to:date_local:HH}` could be added, not to break current three part formatting.

In this case `date` will be in UTC completely, and `date_local` in browser time.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
